<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleManager</class>
 <widget class="QWidget" name="RoleManager">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnAddRole">
         <property name="text">
          <string>添加角色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEditRole">
         <property name="text">
          <string>编辑角色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeleteRole">
         <property name="text">
          <string>删除角色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRefresh">
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>权限设置：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treePermissions">
       <column>
        <property name="text">
         <string>权限项</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSavePermissions">
       <property name="text">
        <string>保存权限设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 